### ISSUE TITLE
NumberFormatException: null - for additional_info.original_amount

### ISSUE THREAD

**Issue by JohannesHome** [figo-connect/java-figo#43]
I'm getting a NumberFormatException when deserializing the `TransactionsResponse` response from `/rest/accounts/${id}/transactions` for some users. 

`AdditionalTransactionInfo` expects BigDecimal. But the response I'm getting looks like this:

```json
      "additional_info": {
        "compensation_amount": "6,10",
        "original_amount": "575,40"
      }
```

I assume this is a bug?

**Comment by berend** [comment]
Yes, that looks like a bug. Looks like a `String` to `BigDecimal` is missing.

**Comment by berend** [comment]
Sorry for the delay, debugging this revealed some differences from bank to bank while parsing this. Only solution we have so far is rolling back to sdk version `1.5.1`

**Comment by berend** [comment]
Also the bug is not a missing cast from string to decimal, but the value containing a comma `,` instead of a dot `.`

**Comment by berend** [comment]
fixed with #44

**Comment by berend** [comment]
Thanks for the feedback, I will look into this!

**Comment by berend** [comment]
We have some internal dependencies for this bug here. Some of our bank connector return `,` instead of `.`, we are fixing this on the API side, not on the SDK. This should only affect a small number of banks. 

I am leaving this open until this is completely fixed.
